lib/OnboardingMessageProvider.jsm

```diff
@@ -534,8 +537,9 @@ const ONBOARDING_MESSAGES = async () => [
       body:
         "The New Enhanced Tracking Protection, gives you the best level of protection and performance. Discover how this version is the safest version of firefox ever made.",
       link_text: "Learn more on our blog",
-      link_url: "https://blog.mozilla.org/",
+      cta_url: "https://blog.mozilla.org/",
     },
+    priority: 1,
```

Comment:
Why was this needed?

Reply:
oh right. I should've sorted by date instead of adding this 😝